py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -254,6 +254,20 @@ def _predicate(driver):
     return _predicate
 
 
+def text_to_be_present_in_element_non_empty(locator):
```

Comment:
I need to discuss with those in the core team; I am not sure our current stance on adding additional expected_conditions - and we should update the docs to be clear (if we prefer them to be more user defined in their own projects) - will report back shortly

Reply:
Ok, thanks, I'll wait for decision